docs/conf.py

```diff
@@ -129,6 +129,7 @@
         },
     ],
     "header_links_before_dropdown": 4,
+    "header_dropdown_text": "Other",
```

Comment:
I would not change the one from the docs (which should show a vanilla version of the documetnation as much as possible). Instead could you create a test in the test suits ? let us know if you don't understand how to override parameters.

Reply:
Hope I got this right: [f0cb271](https://github.com/pydata/pydata-sphinx-theme/pull/1423/commits/f0cb271b9119fc9a75806d9d73f7a82af6deac3d)  =)